apache/zeppelin
# [ZEPPELIN-2699] Helium REST API bug-fixes. Also updated documentation

### What is this PR for?
Fixing bug on enabling Helium plugins (artifact is empty, not null) when sending curl command.
After noticing out-of-date documentation, I updated it
Also noticed issues on missing try/catch expressions and missing info messages on enable/disable packages.

### What type of PR is it?
[Bug Fix | Documentation | Refactoring]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2699

### How should this be tested?
Run "curl -X POST http://localhost:8080/api/helium/enable/{packageName}"
Check that the package gets enabled

### Questions:
* Does the licenses files need update? N
* Is there breaking changes for older versions? N
* Does this needs documentation? Y (Done)
